parrots.yaml

```diff
@@ -466,4 +466,10 @@
   tip: Looks kinda sus
 - gif: chefparrot.gif
   name: Chef parrot
-  tip: Cooking parrot
\ No newline at end of file
+  tip: Cooking parrot
+  gif: appleparrots.gif
+  name: apple parrot
```

Comment:
No offense, but that really looks like a pear to me.  Can we change the gif to `pear-parrots.gif` and name to `Pear Parrots`?  Is there a story behind this gif?

Reply:
lol its been updated to pear. Accept PR